crypto/s2n_ecc_evp.c

```diff
@@ -267,22 +269,22 @@ int s2n_ecc_evp_compute_shared_secret_as_server(struct s2n_ecc_evp_params *ecc_e
 #endif
 
     return s2n_ecc_evp_compute_shared_secret(ecc_evp_params->evp_pkey, peer_key,
-                                             ecc_evp_params->negotiated_curve->iana_id, shared_key);
-
+            ecc_evp_params->negotiated_curve->iana_id, shared_key);
 }
 
 int s2n_ecc_evp_compute_shared_secret_as_client(struct s2n_ecc_evp_params *ecc_evp_params,
-                                            struct s2n_stuffer *Yc_out, struct s2n_blob *shared_key) {
-
-    DEFER_CLEANUP(struct s2n_ecc_evp_params client_params = {0}, s2n_ecc_evp_params_free);
+        struct s2n_stuffer *Yc_out, struct s2n_blob *shared_key)
+{
+    DEFER_CLEANUP(struct s2n_ecc_evp_params client_params = { 0 }, s2n_ecc_evp_params_free);
 
     POSIX_ENSURE_REF(ecc_evp_params->negotiated_curve);
     client_params.negotiated_curve = ecc_evp_params->negotiated_curve;
     POSIX_GUARD(s2n_ecc_evp_generate_own_key(client_params.negotiated_curve, &client_params.evp_pkey));
     S2N_ERROR_IF(client_params.evp_pkey == NULL, S2N_ERR_ECDHE_GEN_KEY);
 
     if (s2n_ecc_evp_compute_shared_secret(client_params.evp_pkey, ecc_evp_params->evp_pkey,
-                                          ecc_evp_params->negotiated_curve->iana_id, shared_key) != S2N_SUCCESS) {
```

Comment:
Interesting. I don't have any objection here, but it did reflow this line. Even with ColLimit 0.

Reply:
Is that the reason? It also did it [here](https://github.com/aws/s2n-tls/blob/bf7c834191da0818629669a33fb9c43327f4e27a/crypto/s2n_ecc_evp.c#L496-L499). Neither looks particularly readable.